tests/lib/snaps/apply-config-hook-set/meta/hooks/apply-config

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+echo "Setting a configuration value"
+if ! snapctl set baz=qux; then
```

Comment:
Well, I use that one to verify that snapctl works as expected (see the snapctl-get-set spread test). These test the interaction between the snap set/get and snapctl get/set commands, so a little different.